artemis-journal/src/main/java/org/apache/activemq/artemis/core/journal/impl/JournalImpl.java

```diff
@@ -108,6 +115,7 @@
     *
     * */
    public static final double UPDATE_FACTOR;
+   public static final String BKP = ".bkp";
```

Comment:
If its going to another directory do we really need or even want a suffix on it?

Reply:
This is for the current journal file.

I rename it to BKP, in case the node crashes before the copy is finished, when restarted the copy is finished and the journal name restored. This is for the pending files to be copied from the journal to the BKP area.

Reply:
Thanks